base/commands/list/list_destroy.go

```diff
@@ -0,0 +1,63 @@
+package _list
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	"github.com/hazelcast/hazelcast-commandline-client/errors"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/prompt"
+)
+
+type ListDestroyCommand struct{}
+
+func (mc *ListDestroyCommand) Init(cc plug.InitContext) error {
+	long := `Destroy a List
+
+This command will delete the List and the data in it will not be available anymore.`
+	short := "Destroy a List"
+	cc.SetCommandHelp(long, short)
+	cc.AddBoolFlag(clc.FlagAutoYes, "", false, false, "skip confirming the destroy operation")
+	cc.SetCommandUsage("destroy [flags]")
+	return nil
+}
+
+func (mc *ListDestroyCommand) Exec(ctx context.Context, ec plug.ExecContext) error {
+	lv, err := ec.Props().GetBlocking(listPropertyName)
+	if err != nil {
+		return err
+	}
+	autoYes := ec.Props().GetBool(clc.FlagAutoYes)
+	if !autoYes {
+		prompt := prompt.New(ec.Stdin(), ec.Stdout())
```

Comment:
```suggestion
		p := prompt.New(ec.Stdin(), ec.Stdout())
```
import and variable names should be different.